openedx/features/learner_profile/templates/learner_profile/learner-achievements-fragment.html

```diff
@@ -0,0 +1,63 @@
+## mako
+
+<%page expression_filter="h"/>
+
+<%namespace name='static' file='/static_content.html'/>
+
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+
+<div class="learner-achievements">
+    <h3 class="u-field-title">Course Certificates</h3>
+    % if course_certificates:
+        % for certificate in course_certificates:
+            <%
+            certificate_url = certificate['download_url']
+            course = certificate['course']
+            %>
+            % if certificate_url:
+                <a href="${certificate_url}" target="_blank">
+                    <div class="card certificate-card mode-${certificate['type']}">
+                        <div class="card-logo">
+                            <h4 class="sr-only">
+                                ${Text(_('{course_mode} Certificate')).format(
+                                    course_mode=certificate['type'],
+                                )}
+                            </h4>
+                        </div>
+                        <div class="card-content">
+                            <div class="card-supertitle">${course.display_org_with_default}</div>
+                            <div class="card-title">${course.display_name_with_default}</div>
```

Comment:
@andy-armstrong: The sandbox examples made it unclear what was being listed here, so I wrote a comment on HipChat.  It seems these are just the course name, so I think it is good.